composer.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "drupal/commerce_bluesnap",
+  "type": "drupal-module",
+  "description": "Provides Drupal Commerce integration with the BlueSnap Payment Gateway.",
+  "keywords": ["Drupal"],
+  "homepage": "https://www.drupal.org/project/commerce_bluesnap",
+  "license": "GPL-2.0+",
+  "authors": [
+    {
+      "name": "shabana.navas",
+      "homepage": "https://www.drupal.org/u/shabananavas",
+      "role": "Maintainer"
+    },
+    {
+      "name": "krystalcode",
+      "homepage": "https://www.drupal.org/u/krystalcode",
+      "role": "Co-maintainer"
+    }
+  ],
+  "support": {
+    "issues": "https://www.drupal.org/project/issues/commerce_bluesnap",
+    "source": "http://cgit.drupalcode.org/commerce_bluesnap"
```

Comment:
**CHANGE REQUEST**

After the switch to GitLab that should be `https://git.drupalcode.org/project/commerce_bluesnap.git`.